test/parallel_api/numeric/numeric.ops/inclusive_scan_by_segment.pass.cpp

```diff
@@ -53,31 +73,57 @@ struct test_inclusive_scan_by_segment
         }
     }
 
-    template <typename Iterator1, typename Iterator2, typename Size>
+#ifdef DUMP_CHECK_RESULTS
+    template <typename Iterator, typename Size>
+    void display_param(const char* msg, Iterator it, Size n)
+    {
+        using DifferenceType = typename DifferenceTypeT<Iterator>::DifferenceType;
+
+        std::cout << msg;
+        for (DifferenceType i = 0; i < n; ++i)
```

Comment:
Actually, we can use Size type instead of additional DifferenceType.

Reply:
Fixed